### PR TITLE
Component Console html

### PR DESCRIPTION
**Pull request by rena-deane**
- added html to Console
- considering a refactor to split the sections of the console into their own little components

### CODE REVIEW

File: client/components/Console.js [hunk 1]
```diff
@@ -11,9 +11,71 @@ class Console extends React.Component {
...
+
+        </div>
+
+        <h3 className='console-headers'>ADD COMMENT</h3>
+        <div className='add-comment'>
+          <input type='text' className='console-comment' id='add-comment'/>
+          <button type='submit' className='submit button' id='submit-comment'>+ Submit</button>
+        </div>
+
+        <h3 className='console-headers'>ADD COMMENT</h3>
```
Comment (kristy-mj) [inline client/components/Console.js]: Edit comment?

File: client/components/Console.js [hunk 2]
```diff
@@ -11,9 +11,71 @@ class Console extends React.Component {
...
+        <div className='edit-comment'>
```
Comment (kristy-mj) [inline client/components/Console.js]: Edit comment container/wrapper?

File: client/components/Console.js (hunk 1 above)
Comment (rena-deane) [inline client/components/Console.js]: done

**Comment by rena-deane** [comment]
waffle #38

**Pull request merged**